src/hooks/useGetAssetIcon.tsx

```diff
@@ -4,7 +4,7 @@ import USDC from '../assets/coins/USDC.png';
 import USDT from '../assets/coins/USDT.svg';
 import USDC_POLYGON from '../assets/coins/USDC_POLYGON.svg';
 import USDT_POLYGON from '../assets/coins/USDT_POLYGON.svg';
-import ASSETHUB_POLYGON from '../assets/coins/ASSETHUB_POLYGON.svg';
+import ASSETHUB_POLYGON from '../assets/coins/USDC_ASSETHUB.svg';
```

Comment:
Let's also rename the import 😅 
```suggestion
import USDC_ASSETHUB from '../assets/coins/USDC_ASSETHUB.svg';
```